plugins/pubIds/urn/URNPubIdPlugin.php

```diff
@@ -402,6 +403,9 @@ public function addPublicationFormFields($hookName, $form)
             $pattern = $this->getSetting($form->submissionContext->getId(), 'urnPublicationSuffixPattern');
         }
 
+        // Load the checkNumber.js file that is required for this field
+        $this->addJavaScript(Application::get()->getRequest(), TemplateManager::getManager(Application::get()->getRequest()));
```

Comment:
This doesn't appear to do anything.

Reply:
this adds checkNumber.js -- s. function addJavaScript below...

Reply:
Ahh sorry, got this confused with the generic method. :+1: